Account for rowspan in table `headingColumns`. (#19366)
ckeditor/ckeditor5: .changelog/20251114122902_ck_14826.md

---
type: Fix
scope:
  - ckeditor5-table
closes:
  - https://github.com/ckeditor/ckeditor5/issues/14826
communityCredits:
  - bendemboski
---

> Fixed an issue where table cells with `rowspan` attributes were not properly accounted for when computing `headingColumns` during upcast. This caused header columns to be incorrectly stripped when using `editor.setData()` on tables with merged cells in the heading columns.

Reviewer: Maybe we could be less technical here, for example:

```suggestion
Fixed an issue where tables with merged cells (rowspan) in header columns were not handled correctly.
```

Or something similar, WDYT?
